inductiva/fluids/scenarios/wind_terrain/wind_terrain.py

```diff
@@ -0,0 +1,205 @@
+"""Wind terrain scenario for air flowing over complex terrains."""
+
+from functools import singledispatchmethod
+import os
+import shutil
+import tempfile
+from typing import List, Optional
+import uuid
+
+import numpy as np
+
+import inductiva
+from inductiva import fluids
+
+SCENARIO_TEMPLATE_DIR = os.path.join(inductiva.utils.templates.TEMPLATES_PATH,
+                                     "wind_terrain")
+OPENFOAM_TEMPLATE_INPUT_DIR = "openfoam"
+FILES_SUBDIR = "files"
+COMMANDS_TEMPLATE_FILE_NAME = "commands.json.jinja"
+TERRAIN_FILENAME = "terrain.stl"
+
+class WindOverTerrain(inductiva.scenarios.Scenario):
+    """Wind Flowing over complex terrain scenario.
+
+    This simulation scenario models the steady-state conditions of
+    wind flowing over complex terrain (e.g., mountains, valleys, etc.).
+    
+    The terrain is modeled through a 2D surface in a 3D world and defined
+    through a mesh. The wind is injected through one of the side walls with
+    a certain vectorail velocity and position. Here, the wind is initialized
+    only on a circular region of the wall to simulate gusts of wind.
+
+   Schematic representation of the Wind and Simulation scenario:
+
+         Wind Injection                 2D Representation of Simulation
+     ____________________                     ____________________
+    |                    |                   |                    |
+    |    * *             |                   |                    |
+    |  *     *           |                   |->                  |
+    |  *     *           |                   |->          /\      |
+    |    * *             |                   |         /\/  \     |
+    |____________________|                   |________/______\____|
+
+    This scenario solves the steady-state continuity and momentum equations
+    (time-independent) with the assumption of incompressible flow. 
+    The simulation solves the time-independent equations for several
+    time steps, based on the state of the previous one. The end goal is
+    to determine the steady-state of the system, i.e., where the flow
+    does not change in time anymore.
+
+    This scenarion can be simulation with OpenFOAM.
+    """
+
+    valid_simulators = [fluids.simulators.OpenFOAM]
+
+    def __init__(self,
+                 terrain: inductiva.world.Terrain,
+                 wind_velocity: List[float],
+                 wind_position: Optional[List[float]] = None,
+                 athmosphere_height: float = 300):
+        """Initializes the `WindTerrain` conditions.
+
+        Args:
+            wind_velocity (List): Velocity of the air flow (m/s).
+            wind_position (List): Position of the wind flow (m) above the height
+            of the terrain.
```

Comment:
Is this "above the local height"? Or which height does this refer to?

Reply:
Indentation missing.

Reply:
Confirming this!

Reply:
What exactly do you confirm? That it is "above the local height"? If so, please add the detail to the docstring.

Reply:
I will confirm this before merging, and adequately mention in the docstring.